build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     //Unit testing framework.
     testCompile 'junit:junit:4.12'
+    //testCompile "org.mockito:mockito-core:2.+"
```

Comment:
it's a line we'll need if we use mockito, and I suspect we will be using mockito for testing RSN -- like maybe for the authentication piece.   But it can certainly be removed.  What say you, @tingulfsen @jmartin-sul ?